localization/languages/cs.json

```diff
@@ -81,9 +81,9 @@
     "findMatchesPlural": "%i z %t výsledků",
     /* Focus mode */
     "isFocusMode": "Jste v režimu zaměření.",
-    "closeDialog": "OK", //used as a label for the button that closes the dialog
-    "focusModeExplanation1": "V režimu zaměření nemůžete přepínat záložky nebo měnit dimenze.",
-    "focusModeExplanation2": "Můžete opustit režim zaměření odškrtnutím \"režim zaměření\" v menu zobrazení.",
+    "closeDialog": "Rozumím", //used as a label for the button that closes the dialog
```

Comment:
If I understand correctly, this means "I understand", which makes sense for the focus mode dialog. However, this also gets used for the [about dialog](https://github.com/minbrowser/min/blob/d8ed7acad9230e7fdba4e801560ad9b8221d66e8/main/menu.js#L409) and potentially other dialogs in the future. Does this still make sense in that case?

Reply:
You are right. In other dialogs it doesn't make much sense. Let's use universal "close" instead.